docs/source/working_with_sources.rst

```diff
@@ -119,96 +105,11 @@ To create a satellite source, use the following steps:
-
-To generate a summary report, enter the ``report summary`` command and specify the identifier for the scan job and the format for the output file.
-
-For example, if you want to create the report summary for a scan with the scan job identifier of ``1`` and you want to generate that report in CSV format in the ``~/scan_result.csv`` file, you would enter the following command::
-
-  # qpc report summary --id 1 --csv --output-file=~/scan_result.csv
-
-However, if you want to create the detailed report, you would use the ``report detail`` command.  This command takes the same options as the ``report summary`` command. The output is not deduplicated and merged, so it contains all facts from each source. For example, to create the detailed report for a scan with the scan job identifier ``1``, with CSV output in the ``~/scan_result.csv`` file, you would enter the following command::
-
-  # qpc report detail --id 1 --csv --output-file=~/scan_result.csv
-
-Pausing and Restarting a Scan
------------------------------
-As you use Quipucords, you might need to stop a currently running scan. There might be various business reasons that require you to do this, for example, the need to do an emergency fix due to an alert from your IT health monitoring system or the need to run a higher priority scan if a lower priority scan is currently running.
-
-When you stop a scan by using the ``scan pause`` command, you can restart that same scan by using the ``scan restart`` command. To pause and restart a scan, use the following steps:
-
-1. Make sure that you have the scan job identifier for the currently running scan. To obtain the scan job identifier, see the information in `Showing Scan Job Results for an Active Scan`_.
-
-2. Enter the command to pause the scan job. For example, if the scan job identifier is ``1``, you would enter the following command::
-
-    # qpc scan pause --id 1
-
-3. When you are ready to start the scan job again, enter the command to restart the scan. For example, to restart scan ``1``, you would enter the following command::
-
-    # qpc scan restart --id 1
-
-
-Logging out of the Quipucords Server
-------------------------------------
-When you log in to the server, the command retrieves a token that is used for authentication with subsequent command line interface commands. That token expires daily. In addition, the token is removed when you log out of the server.
+Editing, Listing, and Clearing Credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Beyond creating credentials for use with sources you can perform several other operations. You can edit a credential for cases where passwords or passphrases need to be updated. You can list and filter credentials. For credentials that are no longer needed you can they can be removed. These interactions are described in the `Credentials section <man.html#credentials>`_ of the Command Line Usage.
```

Comment:
For credentials that are no longer needed you can they can --> Credentials that are no longer needed can be removed.

Reply:
done